src/assets/src/services/api.ts

```diff
@@ -171,3 +171,15 @@ export const searchQueue = async (term: string) => {
     await handleErrors(resp);
     return await resp.json() as QueueAttendee[];
 }
+
+export const changeAgenda = async (meeting_id: number, agenda: string) => {
+    const resp = await fetch(`/api/meetings/${meeting_id}/`, {
+        method: "PATCH",
+        headers: getPatchHeaders(),
+        body: JSON.stringify({
+            agenda: agenda,
+        }),
+    });
+    await handleErrors(resp);
+    return await resp.json();
+}
```

Comment:
This might have already been there, but can you add a newline to end of this file?